portfolio/src/main/webapp/script.js

```diff
@@ -28,6 +28,33 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function commentHandler(){
+ fetch('/load-comments').then(response => response.json()).then((Comment) =>{
+     const commentListElement = document.getElementById('comment-list');
+     console.log("before the for loop");
+     Comment.forEach((text) => {
+         console.log("In the loop");
+         commentListElement.appendChild(newComment(text.comment));
+     })
+ }); 
+    
+}
+
+function newComment(text){
+    const commentElement = document.createElement('li');
+    commentElement.className = "comment";
+
+    commentElement.innerText = text;
+
```

Comment:
this white space seems potentially unnecessary

Reply:
+1